### PR TITLE
new article with tips for PowerShell users

### PR DESCRIPTION
**Pull request by dbradish-microsoft**
This PR completes the following:
1. rename use-cli-effectively.md to use-azure-cli-successfully.md, update links and add redirect
2. **add new article use-azure-cli-successfully-in-powershell.md**
3. slight reorganization of the TOC and Main landing page

### CODE REVIEW

File: docs-ref-conceptual/use-azure-cli-successfully.md
Comment (daphnemamsft) [inline docs-ref-conceptual/use-azure-cli-successfully.md:1]: Can you define what "piping" is?

File: docs-ref-conceptual/use-azure-cli-successfully-in-powershell.md [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+Some Azure CLI parameter values, such as Azure resource tags, require key:value pairs. If your `key` or `value` contains a space or special character, the Bash and PowerShell syntax are not always the same.
+
+See [Create tags to practice quoting differences](./get-started-tutorial-2-environment-syntax.md#create-tags-to-practice-quoting-differences) in the [Learn to use the Azure CLI](./get-started-tutorial-0-before-you-begin.md) tutorial. This tutorial step gives examples for Bash, PowerShell and Cmd for the following key:value pair scenarios:
+
+* spaces
+* empty values
+* special characters
+* variables
+
+## Error handling for Azure CLI in PowerShell
```
Comment (daphnemamsft) [inline docs-ref-conceptual/use-azure-cli-successfully-in-powershell.md:399]: Is error handling something fundamental? As in, would this information be important for a beginner to know? Or is it a "nice to know" or "more info" kind of topic? The rest of the article seems set up for beginners but the error handling portion seems a bit more advanced?

Comment (dbradish-microsoft) [reply docs-ref-conceptual/use-azure-cli-successfully-in-powershell.md:399]: @daphnemamsft , right now, this article is intended for a catch-all of tips for customers running the Azure CLI in a PowerShell environment.

**Comment by dbradish-microsoft** [comment]
Merging PR until it gets any larger!

**Review by Moazzem-Hossain-pixel — COMMENTED** [review]
Need support

**Pull request merged**